source/agora/common/Transaction.d

```diff
@@ -259,48 +259,26 @@ public Transaction newCoinbaseTX (PublicKey address, Amount value = 0)
 *******************************************************************************/
 
 version (unittest)
-public Transaction[] getChainedTransactions (Transaction root, size_t count, KeyPair key_pair, size_t period = 1)
+public Transaction[] getChainedTransactions (Transaction root, size_t count,
```

Comment:
Commit fed184f52e4d4b1e19811c9cc23d6f78e588ae08 "Revert changes to getChainedTransactions()".
You delete a bunch of code but don't change any test. Does that mean that code is unused, or do the tests fail ?

Reply:
Yes this is the non-atomic part. But if I want to make it atomic then I have to squash 3 commits together, which kind of sucks.

I wish there was a way to make "sub-atomic" commits.